src/gameStateService/data-access-objects.ts

```diff
@@ -101,4 +103,13 @@ export const createStonePositionsDAO = ({
 	return randomizedStonePositions;
 }
 
+export const createLevelEndDataDAO = (gameState) => {
+  return {
+		starCount: gameState.levelEndData.starCount,
+		currentLevel: gameState.levelEndData.currentLevel,
+		isTimerEnded: gameState.levelEndData.isTimerEnded,
+		data: gameState.data
```

Comment:
These are common data to all scenes.

Can you add a todo to organize this later?